mixers/BaseMix.sc

```diff
@@ -50,6 +69,17 @@ BaseMix : Object {
 		server.doWhenBooted({waitForServer.value});
 	}
 
+	// sendSynthDefs method sends definitions from a file to the server for use in audio mixing
+	sendSynthDefs { | filename |
```

Comment:
TODO: add some "do once" logic here to ensure that each synthdef is only sent to the server once (maybe using `File.exists` or something?). We should make this a bool parameter like `useCache` so that it can be disabled for dev and testing purposes.